### PR TITLE
DP-4325 Final Patterns Documentation

### PR DESCRIPTION
**Pull request by gender404**
Updated Patterns used on the Homepage, Error Page and Transition Page.  A few other Patterns that were missed or that weren't used on a page were updated as well.

Atoms:
* Arrow Button (fixed CSS)

Molecules:
* Arrow Nav (switched link to use title)
* Divided Inputs
* Search Banner Form
* Tooltip

Organisms:
* Callout
* Helpful Links
* Location Listing
* Page Flipper
* Rich Text (enhanced to include an optional learn more link)
* Search Banner
* Error Page
* Image Credit
* Transition Page

Templates:
* Narrow Template
* Stacked Row (updated docs and added header and footer blocks)

Pages:
* Error-403
* Error-404
* Error-500
* Homepage
* Board Decision (updated docs)
* Transition

### CODE REVIEW

File: styleguide/source/_patterns/01-atoms/01-buttons/arrow-button.md [hunk 1]
```diff
@@ -1,21 +1,17 @@
...
-    type: string/required
-  "info": 
-    type: string/required - used for screen readers
-  "direction": 
-    type string/optional
+arrowButton: {
+  href: 
+    type: string (url) / required
+  info: 
+    type: string (adds more description to the link) / optional
```
Comment (jesconstantine) [inline styleguide/source/_patterns/01-atoms/01-buttons/arrow-button.md]: I think for accessibility, this variable should be required - thoughts?

File: styleguide/source/_patterns/02-molecules/arrow-nav.twig [hunk 2]
```diff
@@ -1,8 +1,9 @@
 {% set arrowButton = arrowNav.arrowButton %}
+  
 <section class="ma__arrow-nav {{ arrowButton.direction ? 'ma__arrow-nav--' ~ arrowButton.direction : '' }}">
   {% include "@atoms/01-buttons/arrow-button.twig" %}
```
Comment (jesconstantine) [inline styleguide/source/_patterns/02-molecules/arrow-nav.twig:4]: Is there a way to relate the arrow button to the arrow nav title/link content?  Curious how a screenreader would announce this here...

File: styleguide/source/_patterns/03-organisms/by-author/helpful-links.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+### Description
+List of links that can be rendered stack or as two columns
```
Comment (jesconstantine) [inline styleguide/source/_patterns/03-organisms/by-author/helpful-links.md:2]: Could this pattern be a variant of a link list organism?

**Review by jesconstantine — APPROVED** [review]
Thank you @legostud !  I have some quick thoughts / comments but I think this one is good to go as well!

File: styleguide/source/_patterns/03-organisms/by-author/helpful-links.md (hunk 3 above)
Comment (jesconstantine) [reply styleguide/source/_patterns/03-organisms/by-author/helpful-links.md:2]: Nevermind - I see this is done in a later PR! 👍

File: styleguide/source/_patterns/01-atoms/01-buttons/arrow-button.md (hunk 1 above)
Comment (gender404) [reply styleguide/source/_patterns/01-atoms/01-buttons/arrow-button.md]: true

**Pull request merged**